pkg/nodes/drain.go

```diff
@@ -0,0 +1,67 @@
+package nodes
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubectl/pkg/drain"
+)
+
+// NewDrainHelperBuilder builds drain Helper that contains parameters to control the behaviour of drain.
+func NewDrainHelperBuilder(
```

Comment:
I think:
Drain/Cordon/Uncordon functions have to a part of node.Builder struct.
Please add these methods as part of Builder in node.go file